### ISSUE TITLE
Implement Email Authentication for signup!

### ISSUE THREAD

**Issue by sakshi-dhamija** [secrashi/Connect-Intellect#5]

**Comment by Manoramsharma** [comment]
@secrashi , I would like contribute to the given issue. Also email auth or gmail auth what exactly to be done ?

**Comment by sakshi-dhamija** [comment]
Email Auth. Assigning it to you.

**Comment by Manoramsharma** [comment]
@secrashi, already there is signup controller in server folder , so what exactly I required to do ?

**Comment by sakshi-dhamija** [comment]
Yeah, sign-up/ login functionality is available. The task is authenticating. For example: when we signup for a website, we get a mail for confirmation if that's our Id.
